package-lock.json

```diff
@@ -6817,108 +6817,6 @@
       "integrity": "sha512-9rTIZ4ZjWwalCPiaY+kPiALLfOKgAz5CTi/Zb1L+qSZ8PH3zVo1T8JcgXIIqg1iM3pZ6hF+n9xO5r2jZ/SF+jg==",
       "dev": true
     },
-    "node_modules/devtools/node_modules/debug": {
```

Comment:
Do you know why these changes in the `package-lock.json` popped up? If not, can we exclude them from the PR?

Reply:
Excluded :)